analytics/integration_test/src/integration_test.cc

```diff
@@ -341,4 +341,70 @@ TEST_F(FirebaseAnalyticsTest, TestSetConsent) {
   did_test_setconsent_ = true;
 }
 
+TEST_F(FirebaseAnalyticsTest, TestDefaultEventParametersUsage) {
```

Comment:
Rename this test TestSetDefaultEventParameters and the following test TestClearDefaultEventParameters.